README.md

```diff
@@ -12,8 +12,8 @@ following new functionality:
 * Controllers to provision these resources in GitHub based on the users desired
   state captured in CRDs they create
 * Implementations of Crossplane's [portable resource
-  abstractions](https://crossplane.io/docs/master/running-resources.html),
-  enabling GitHub resources to fulfill a user's general need for cloud services
+  abstractions](https://github.com/crossplane/provider-github), enabling GitHub
```

Comment:
the other PRs linked to https://crossplane.io/docs/master/concepts.html

Reply:
Oops, sorry about that! Copy paste error :upside_down_face: